src/main/scala/filter2.scala

```diff
@@ -1,19 +1,18 @@
 package era7bio.db.rna16s
 
 import era7bio.db._, csvUtils._, collectionUtils._
-
 import ohnosequences.fastarious.fasta._
 import ohnosequences.statika._
-
 import com.github.tototoshi.csv._
 import better.files._
 
-case object filter2 extends FilterDataFrom(filter1)() {
+// TODO I don't understand what this is trying to do, and how.
```

Comment:
I want first a readable global description of what `filterData` does. I'm merging this anyway, and we will do this in a separate PR.